Makefile

```diff
@@ -209,18 +209,19 @@ go-build: pilot istioctl pilot-agent sidecar-initializer mixs mixc node-agent is
 # Target: go test
 #-----------------------------------------------------------------------------
 
-.PHONY: go-test localTestEnv testApps
+.PHONY: go-test localTestEnv test-bins
 
 GOTEST_PARALLEL ?= '-test.parallel=4'
 GOTEST_P ?= -p 1
 GOSTATIC = -ldflags '-extldflags "-static"'
 
-testApps:
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/server istio.io/istio/pilot/test/server
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/client istio.io/istio/pilot/test/client
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/eurekamirror istio.io/istio/pilot/test/eurekamirror
+test-bins:
+	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/pilot-test-server istio.io/istio/pilot/test/server
```

Comment:
missing LDFLAGS in addition to CGO_ENABLED

Reply:
That doesn't seem to matter. Tests are working.. Since we are not shipping these, is there any big issue if we dont have ldflags?

Reply:
yes, it means we don't really have statically compiled binaries and there might be host OS influence.